src/InformaticsGateway/Services/Http/SourceAeTitleController.cs

```diff
@@ -88,6 +88,32 @@ public async Task<ActionResult<SourceApplicationEntity>> GetAeTitle(string name)
             }
         }
 
+        [HttpGet("/getbyaetitle/{aeTitle}")]
```

Comment:
Do you think we can simplify this to `/aetitle/{ae-title}`?

Reply:
sure, fair enough